From c28dfa55743d3389c2ba0cb63c390a57c812a8c0 Mon Sep 17 00:00:00 2001
From: v1s1t0r <v1s1t0r.1s.h3r3@gmail.com>
Date: Tue, 30 May 2017 08:55:14 +0200
Subject: [PATCH] remove-update

---
 airgeddon.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/airgeddon.sh b/airgeddon.sh
index 6e1b22f..e9557af 100755
--- a/airgeddon.sh
+++ b/airgeddon.sh
@@ -11,7 +11,7 @@
 debug_mode=0
 
 #Enabled 1 / Disabled 0 - Auto update feature (it has no effect on debug mode) - Default value 1
-auto_update=1
+auto_update=0
 
 #Enabled 1 / Disabled 0 - Auto change language feature - Default value 1
 auto_change_language=1
-- 
2.11.0

